build/rpmbuild_internal.h

```diff
@@ -19,6 +19,7 @@
 #undef HTDATATYPE
 
 #define ALLOWED_CHARS_NAME ".-_+%{}"
+#define ALLOWED_CHARS_NAME_FIRST "_%{}"
```

Comment:
Ditto here, ALLOWED_FIRSTCHARS_NAME will be more obvious to the reader and easier to grep for.